operator/v1/types_console.go

```diff
@@ -143,8 +143,48 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomLogoThemeFile defines a configuration based on theme types for the console UI logo.
+type CustomLogoThemeFile struct {
```

Comment:
This seems like a good time to generalize custom logos to support favicons as well. IIRC we've had that request before. So maybe there are properties like `type: {Masthead|Favicon}` and `theme: {Light|Dark}`?

cc @rhamilto

Reply:
That's good thinking, @spadgett.  https://issues.redhat.com/browse/CONSOLE-4061 and https://issues.redhat.com/browse/CONSOLE-4062 are the stories for the custom favicon.